tidb-cloud/tidb-cloud-billing.md

```diff
@@ -188,4 +188,52 @@ If you have agreed with our sales on a contract and received an email to review
 3. Click **Contract**. The contract list is displayed.
 4. Click **Download**, **Accept**, or **Reject** according to your need.
 
-To learn more about contracts, feel free to [contact our sales](https://www.pingcap.com/contact-us/).
\ No newline at end of file
+To learn more about contracts, feel free to [contact our sales](https://www.pingcap.com/contact-us/).
+
+## Billing from AWS Marketplace or Google Cloud Marketplace
+
+If you are the owner or billing administrator of your organization, you can link your TiDB Cloud account to an AWS billing account or Google Cloud billing account. Otherwise, skip this section.
+
+If you are new to TiDB Cloud and do not have a TiDB Cloud account, you can sign up for a TiDB Cloud account through [AWS Marketplace](https://aws.amazon.com/marketplace) or [Google Cloud Marketplace](https://cloud.google.com/marketplace), and pay the usage via the AWS or GCP billing account.
+
+If you already have a TiDB Cloud account, you can link the account to an AWS or GCP billing account.
+
+The following sections introduce how to link your TiDB Cloud account to an AWS or GCP billing account to TiDB Cloud. For more details about signing up through Marketplaces, refer to [Create a TiDB Cloud account](/tidb-cloud/create-tidb-cluster.md#step-1-create-a-tidb-cloud-account).
+
+### Link to an AWS billing account
+
+To link your TiDB account to an AWS billing account, take the following steps:
+
+1. Open the [AWS Marketplace page](https://aws.amazon.com/marketplace), search for `TiDB Cloud` and select **TiDB Cloud** in the search results.
+
+2. On the TiDB Cloud product page, click **Continue to Subscribe**.
+
+3. On the TiDB Cloud order page, click **Subscribe**, and then click **Set Up your Account** to go to the TiDB Cloud sign-up page.
+
+4. On the top notification block of the TiDB Cloud sign-up page, click **Sign in** to link TiDB Cloud to your AWS account.
+
```

Comment:
5. Sign in with your TiDB Cloud account. The **Link to Your AWS Billing Account** page is displayed.